src/main/java/org/itsallcode/openfasttrace/importer/legacytag/LegacyTagImporterFactory.java

```diff
@@ -81,7 +83,7 @@ public Importer createImporter(final Path path, final Charset charset,
         if (!config.isPresent())
         {
             final List<String> patterns = this.config.get().getPathConfigs().stream() //
-                    .map(PathConfig::getPattern) //
+                    .map(PathConfig::getDescription) //
```

Comment:
Variable name says "patterns" yet you see to be collecting "descriptions". Please unify.

Reply:
:heavy_check_mark: